apps/snitch_core/test/data/schema/wishlist_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Snitch.Data.Schema.WishListItemTest do
+  use ExUnit.Case
+  use Snitch.DataCase
+  import Snitch.Factory
+
+  alias Snitch.Data.Schema.WishListItem
+
+  setup do
+    user = insert(:user)
+    [user: user]
+  end
+
+  setup :variants
+
+  @tag variant_count: 1
+  test "create a new entry", context do
+    {user, variant} = prepare_user_variant(context)
+    params = %{user_id: user.id, variant_id: variant.id}
+    %{valid?: valid} = changeset = WishListItem.create_changeset(%WishListItem{}, params)
+    assert valid
+    assert {:ok, _} = Repo.insert(changeset)
+  end
+
+  @tag variant_count: 1
+  test "create fails for duplicate user, variant pair", context do
```

Comment:
Don't repeat tests. PLEASE. Try to not hit the repo in schema tests.
This is covered in the model tests.

Reply:
Ok, no repetitions.